src/MagnumPlugins/TinyGltfImporter/TinyGltfImporter.cpp

```diff
@@ -2137,7 +2137,9 @@ Containers::Optional<TextureData> TinyGltfImporter::doTexture(const UnsignedInt
             minFilter = SamplerFilter::Linear;
             mipmap = SamplerMipmap::Linear;
             break;
-        default: CORRADE_INTERNAL_ASSERT_UNREACHABLE(); /* LCOV_EXCL_LINE */
+        default:
+            Error{} << "Trade::TinyGltfImporter::texture(): invalid minFilter" << s.minFilter;
+            return Containers::NullOpt;
```

Comment:
Interesting, for some reason I thought tinygltf *did* check this, which is why the assert. Or maybe I just looked at the enum values and because there was no "invalid" I assumed it handles this gracefully. Ah well.

Thanks for catching this!